packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -1,13 +1,35 @@
-## Recent Activity
+## Recent account activity
+## All strings except title indicate an event that occurred from the user's account
+## These are displayed as a list with the date when the event occured
 
-recent-activity-title = Recent Account Activity
+recent-activity-title-v2 = Recent account activity
 
-recent-activity-account-create = Account was created
-recent-activity-account-disable = Account was disabled
-recent-activity-account-enable = Account was enabled
-recent-activity-account-login = Account initiated login
-recent-activity-account-reset = Account initiated password reset
-recent-activity-emails-clearBounces = Account cleared email bounces
+recent-activity-account-create-v2 = Account created
```

Comment:
Thanks for updating! It is much clearer

Reply:
Thanks @vbudhram! I'll be moving this PR out of draft - would you have time to review?